src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -5,6 +5,16 @@
 configurations_dir = 'asmcnc/apps/drywall_cutter_app/config/configurations'
 cutters_dir = 'asmcnc/apps/drywall_cutter_app/config/cutters'
 
+DEBUG_MODE = True
```

Comment:
Do you want this to be in debug mode on merge?

Reply:
Yeah, will help the other guys with their integration.